src/test/javascript/spec/component/exercises/quiz/manage/quiz-question-list-edit-existing.component.spec.ts

```diff
@@ -446,5 +447,38 @@ describe('QuizQuestionListEditExistingComponent', () => {
             expect(onFilesAddedSpy).toHaveBeenCalledOnce();
             expect(getFileMock).toHaveBeenCalledTimes(3);
         });
+        it('should correctly differentiate between JSON and ZIP files', async () => {
```

Comment:
I'm supprised the client style action does not fail here. Can you add new lines between the tests?